observations/forward_operators/DEFAULT_obs_def_mod.F90

```diff
@@ -542,9 +544,18 @@ subroutine get_expected_obs_from_def_distrib_state(state_handle, ens_size, copy_
    ! if we use them there is no way to compute a consistent posterior.
    ! so the posteriors are always marked as 'failed forward operator'.
    if (use_precomputed_FO) then 
+      if (obs_def%ens_size < ens_size) then
+         write(string1,'(A,1x,I4,1x,A,1x,I4)')'The number of precomputed forward operators (', &
+                      obs_def%ens_size, ') is smaller than the ensemble size of ', ens_size
+         write(string2,*)'observation type '//trim(get_name_for_type_of_obs(obs_def%kind))
+         write(string3,*)'precomputed value(1) ', obs_def%external_FO(1)
+         call error_handler(E_ERR, 'get_expected_obs_from_def', string1, source, &
+                    text2=string2, text3=string3)
+      endif
+
```

Comment:
Checks to make sure there are enough precomputed values for the desired ensemble size.